pkg/dashboards/kubernetes/kubelet/kubelet.go

```diff
@@ -0,0 +1,167 @@
+		panelgroup.PanelsPerLine(2),
+		panels.RunningKubeletStat(datasource, labelMatcher),
+		panels.RunningPodStat(datasource, labelMatcher),
+	)
+}
+
+func withKubeletOperationsQuantile(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Operation Duration 99th quantile",
+		panelgroup.PanelsPerLine(1),
+		panels.OperationDurationQuantile(datasource, labelMatcher),
+	)
+}
+
+func withPodStartRateAndDuration(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Pod Start Rate and Duration",
+		panelgroup.PanelsPerLine(2),
+		panels.PodStartRate(datasource, labelMatcher),
+		panels.PodStartDuration(datasource, labelMatcher),
+	)
+}
+
+func withStorageOperationsAndErrors(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Storage Operations Rate and Errors",
+		panelgroup.PanelsPerLine(2),
+		panels.StorageOperationRate(datasource, labelMatcher),
+		panels.StorageOperationErrorRate(datasource, labelMatcher),
+	)
+}
+
+func withStorageOperationsQuantile(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Storage Operation Duration 99th quantile",
+		panelgroup.PanelsPerLine(1),
+		panels.StorageOperationDuration(datasource, labelMatcher),
+	)
+}
+
+func withCgroupManager(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Cgroup manager",
+		panelgroup.PanelsPerLine(2),
+		panels.CgroupManagerOperationRate(datasource, labelMatcher),
+		panels.CgroupManagerQuantile(datasource, labelMatcher),
+	)
+}
+
+func withPLEGRelist(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("PLEG relist",
+		panelgroup.PanelsPerLine(2),
+		panels.PLEGRelistRate(datasource, labelMatcher),
+		panels.PLEGRelistInterval(datasource, labelMatcher),
+	)
+}
+
+func withPLEGRelistDuration(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("PLEG relist duration",
+		panelgroup.PanelsPerLine(1),
+		panels.PLEGRelistDuration(datasource, labelMatcher),
+	)
+}
+
+func withRPCRate(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("RPC rate",
+		panelgroup.PanelsPerLine(1),
+		panels.RPCRate(datasource, labelMatcher),
+	)
+}
+
+func withRequestDurationQuantile(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
+	return dashboard.AddPanelGroup("Request duration 99th quantile",
+		panelgroup.PanelsPerLine(1),
+		panels.RequestDurationQuantile(datasource, labelMatcher),
+	)
+}
+
+func withKubeletResources(datasource string, clusterLabelMatcher promql.LabelMatcher) dashboard.Option {
+	labelMatchersToUse := []promql.LabelMatcher{
+		promql.ClusterVar,
+		promql.InstanceVar,
+		{
+			Name:  "job",
+			Value: "kubelet",
+			Type:  "=",
```

Comment:
Also @saswatamcode I added here, as I don't see much use outside of this dashboard to add to `globals.go`, but  WDYT?

Reply:
Yup that makes sense. I do have a similar global defined in promql, but this is mostly used as string in promql expressions. For now this is fine